charts/nri-metadata-injection/templates/admission-webhooks/job-patch/job-patchWebhook.yaml

```diff
@@ -42,7 +42,7 @@ spec:
       {{- include "tplvalues.render" ( dict "value" .Values.jobImage.volumes "context" $ ) | nindent 8 }}
```

Comment:
is this needed? couldn't be simplified by using the toYaml func? if so we perhaps can get rid of that extra helper

Reply:
Removed here: [dc327d9](https://github.com/newrelic/k8s-metadata-injection/pull/163/commits/dc327d923093454fa26847c6cb2734d53d73ab5d)